modules/administration-guide/partials/proc_removing-user-data.adoc

```diff
@@ -1,153 +0,0 @@
-[id="removing-user-data_{context}"]
```

Comment:
@themr0c Could you please double-check if this file needs to be deleted? I'm asking for two reasons:
1. The topic seems important.
2. `browse built docs` shows `removing-user-data.adoc` in the menu and
`Unresolved include directive in modules/administration-guide/pages/removing-user-data.adoc - include::partial$proc_removing-user-data.adoc[]`

Reply:
Wow, wrong file deleted. I apologize. I'll blame the kid.